package.json

```diff
@@ -1,1375 +1,1385 @@
 {
-    "name": "vscode-kubernetes-tools",
```

Comment:
Is the only change to this the whitespace?  GitHub doesn't make it clear what's happened but I wonder if it accidentally got changed from spaces to tabs?  It should be spaces with an indent of 4.

Reply:
The indent was changed to tabs. Reverted back to spaces